#By defualt OpenCV looks IPP lib dir in $IPPROOT../lib
# In our setup it is at $IPPROOT/lib
# By B. Hajgato September 17th 2014
--- cmake/OpenCVFindIPP.cmake.orig   2015-12-18 07:02:16.000000000 -0800
+++ cmake/OpenCVFindIPP.cmake       2016-01-25 13:23:15.805798000 -0800
@@ -191,7 +191,7 @@
     _ipp_add_library(m)
 
     if(UNIX)
-      get_filename_component(INTEL_COMPILER_LIBRARY_DIR ${IPP_ROOT_DIR}/../lib REALPATH)
+      get_filename_component(INTEL_COMPILER_LIBRARY_DIR ${IPP_ROOT_DIR}/lib REALPATH)
       if(NOT EXISTS ${INTEL_COMPILER_LIBRARY_DIR})
         get_filename_component(INTEL_COMPILER_LIBRARY_DIR ${IPP_ROOT_DIR}/../compiler/lib REALPATH)
       endif()
